From 3af6ccad75e78ad3e781dd76adceff2a9955022e Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Fri, 4 Sep 2015 18:54:01 +0200
Subject: [PATCH] 2

---
 default.prop | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/default.prop b/default.prop
index d4b3312..6cc2749 100644
--- a/default.prop
+++ b/default.prop
@@ -10,10 +10,10 @@ ro.config.ringtone=Oneplus_tune.ogg
 ro.config.notification_sound=Free.ogg
 ro.config.alarm_alert=Spring.ogg
 persist.sys.usb.config=mtp
-ro.secure=1
+ro.secure=0
 ro.allow.mock.location=0
 ro.debuggable=0
-ro.adb.secure=1
+ro.adb.secure=0
 ro.zygote=zygote64_32
 dalvik.vm.dex2oat-Xms=64m
 dalvik.vm.dex2oat-Xmx=512m
-- 
1.9.1

